sw/device/silicon_creator/lib/base/sec_mmio_unittest.cc

```diff
@@ -194,7 +195,7 @@ TEST_F(SecMmioTest, CheckCount) {
 class SecMmioDeathTest : public SecMmioTest {};
 
 TEST_F(SecMmioDeathTest, Read32OrDieSimulatedFault) {
-  auto deadly_ops = [this] {
+  auto deadly_ops = [] {
```

Comment:
Wanna just inline the lambdas while we're here? :eyes:

Reply:
done